.emacs.d/settings/custom.el

```diff
@@ -8,7 +8,7 @@
 
 ; color theme
 (add-to-list 'custom-theme-load-path (make-plugin-path "color-theme-solarized"))
-(load-theme 'solarized-dark 1)
+(load-theme 'solarized 1)
 (setq solarized-termcolors 256)
```

Comment:
It seems to be, although it's possible something more complicated is going on that I haven't realized yet. 



> On May 19, 2015, at 3:57 PM, Jessica B. Hamrick <notifications@github.com> wrote:
> 
> In .emacs.d/settings/custom.el:
> 
> > @@ -8,7 +8,7 @@
> >  
> >  ; color theme
> >  (add-to-list 'custom-theme-load-path (make-plugin-path "color-theme-solarized"))
> > -(load-theme 'solarized-dark 1)
> > +(load-theme 'solarized 1)
> >  (setq solarized-termcolors 256)
> Is the default the dark theme?
> 
> —
> Reply to this email directly or view it on GitHub.
>